1_html-css/index.html

```diff
@@ -0,0 +1,214 @@
+                </div>
+                <div class="about-me">
+                    <p>In early 2023, I embarked on a transformative journey into the world of frontend development,
+                        igniting a passion for crafting captivating web interfaces. My journey commenced with
+                        self-study, delving into the foundational realms of HTML and CSS. Eager to expand my knowledge
+                        and hone my skills, I enrolled in ReDi School's JavaScript course, delving deeper into the
+                        intricacies of interactive web design. This insatiable drive to learn and grow led me to
+                        continue with a React course, broadening my expertise in crafting dynamic and responsive user
+                        interfaces. As the leaves changed color in October 2023, I took a significant step forward by
+                        joining Hack Your Future, a pivotal juncture in my developmental odyssey, and a testament to my
+                        dedication to becoming a proficient frontend developer</p>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section id="skills" data-aos="fade-up" data-aos-duration="2000" >
+        <div class="skills-title">
+            <p>Explore My</p>
+            <h2 class="big-font">Skills</h2>
+        </div>
+        <div class="skill-bar-container">
+            <div class="skill">
+                <div class="skill-name">HTML</div>
+                <div class="skill-bar">
+                    <div class="skill-level html"></div>
+                    <div>90%</div>
+                </div>
+            </div>
+        </div>
+        <div class="skill-bar-container">
+            <div class="skill">
+                <div class="skill-name">CSS</div>
+                <div class="skill-bar">
+                    <div class="skill-level css"></div>
+                    <div>75%</div>
+                </div>
+            </div>
+        </div>
+        <div class="skill-bar-container">
+            <div class="skill">
+                <div class="skill-name">Bootstrap</div>
+                <div class="skill-bar">
+                    <div class="skill-level bootstrap"></div>
+                    <div>70%</div>
+                </div>
+            </div>
+        </div>
+        <div class="skill-bar-container">
+            <div class="skill">
+                <div class="skill-name">JavaScript</div>
+                <div class="skill-bar">
+                    <div class="skill-level js"></div>
+                    <div>80%</div>
+                </div>
+            </div>
+        </div>
+        <div class="skill-bar-container">
+            <div class="skill">
+                <div class="skill-name">React</div>
+                <div class="skill-bar">
+                    <div class="skill-level react"></div>
+                    <div>50%</div>
+                </div>
+            </div>
+        </div>

```

Comment:
The same thing. Create a container using "ul" and put every skill in "li". In this case, you will reduce the amount of dives😄. 
Bad practice to write text within the <div> tag. The <div> tag is a container element, which allows you to group other elements together and apply styles to them. However, you should use the <p> tag for paragraphs and other long blocks of text, as it provides better semantic meaning and styling options.